projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php

```diff
@@ -136,15 +135,15 @@ function process_anchor_params() {
 
 	// Add Spotify Badge template action.
 	if (
-		$insert_spotify_badge && (
+		$valid_spotify_url && (
```

Comment:
I'm a bit confused by this if statement. I know it is adding the badge for the podcast as a whole rather than a specific episode, but the condition will be true when we have already added the action above. Probably not something to tackle in this PR if it's a problem.

Reply:
I think I read the logic wrong. On a second look it's ok!